src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs

```diff
@@ -11,7 +11,7 @@
    ))
 
 
-(def ^:private initial-rows-per-page 10)
+(def ^:private default-initial-rows-per-page 10)
```

Comment:
I talked about this with Tiffany the other day and she only wanted it changed for the data table _for now_, but I agree that there are a bunch of places where 20 would make more sense